superset/connectors/sqla/models.py

```diff
@@ -1643,43 +1521,6 @@ def _get_series_orderby(
             )
         return ob
 
-    def _normalize_prequery_result_type(
```

Comment:
The parent method seems more complete, handling cases where `column` is not a dict. This is one of the major problems with code duplication, things get fixed in one place but not another. :(